### ISSUE TITLE
Policy loss computes gradients for value network

### ISSUE THREAD

**Issue by pengzhenghao** [cuhkrlcourse/RLexample#12]
This issue exists in `ac-pong-pytorch.py` and `pgb-pong-pytorch`.

I am investigating this problem. It is the possible cause to the training failure.
